published/012-2016-August-24.md

```diff
@@ -0,0 +1,110 @@
+
+Orbit, the decentralized chat app run on top of IPFS, has just integrated uPort identities. There's a writeup on what uPort is [on Medium](https://medium.com/@ConsenSys/uport-the-wallet-is-the-new-browser-b133a83fe73) - essentially, it is a web-based wallet and identity management system. This means that users will be able to use their uPort name and profile picture in Orbit, as well as provision a signing key that will sign Orbit messages. We've been working with [ConsenSys](https://consensys.net/) to make this happen. You can track this [here](https://github.com/haadcode/orbit/issues/37). 
+
+![textui](orbit-textui.gif)
+
+Orbit also now has a [textui](https://github.com/haadcode/orbit-textui). This uses [blessed](https://github.com/chjj/blessed), a curses-like library with a high level terminal interface API for Node.js. This means you can chat to people using Orbit, straight from where you get your work done, in the terminal. Of course, other venues, [like Slack](https://github.com/haadcode/orbit/issues/71), are also in the pipeline.
+
+If you're interested in getting involved with developing Orbit and helping us get there faster, we've been adding a lot more 'Help Wanted' labels to issues that we think could use some work. Look at [these issues](https://github.com/haadcode/orbit/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+sort%3Areactions-%2B1-desc) and dive in. If you have questions, just jump on [the IRC channel](https://webchat.freenode.net/?channels=%23ipfs), or comment on the issues themselves.
+
+### [go-ipfs](//github.com/)
+
+We've updated `go-ipfs` to 0.4.3-rc3 on August 8th. 0.4.2 is still the stable version; however, if you're interested in trying out the release candidates, take a look at the binaries on [dist.ipfs.io](https://dist.ipfs.io/go-ipfs). The 0.4.3-rc comes with a couple of nice enhancements, and improves the performance of storage, bitswap, as well as content and peer routing. It also introduces a handful of new commands and options, and fixes a good bunch of bugs: you can take a look at the full changelog [here](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md).
+
+Moving to go1.7 is in the works, as it will give us performance improvements, `context` has been added into stdlib, and `net.Diallib` will allow our calls to vastly improve resource consumption and downtime. It also adds support for macOS Sierra. [Track that here](https://github.com/ipfs/go-ipfs/issues/3093).
+
+We're working on getting code coverage for all of the packages included in go-ipfs. To follow along or comment, [go here](https://github.com/ipfs/go-ipfs/issues/3053). We also plan on using coveralls to have solid tests for subprojects; follow that [here](https://github.com/ipfs/go-ipfs/issues/3112).
+
+On a related note, @whyrusleeping also udpated `gx deps --tree`, and it's related options, which means that the [gx](https://github.com/whyrusleeping/gx/) package manager we use in IPFS is now easier to use.
```

Comment:
This might read a bit easier if the sub-sentences are turned around "on a related note, gx is easier, because whyrusleeping"